packages/devtools_app/lib/src/flutter/app.dart

```diff
@@ -233,6 +233,13 @@ class OpenSettingsAction extends StatelessWidget {
   }
 }
 
+List<Widget> _header(TextTheme textTheme, String title) {
+  return [
+    Text(title, style: textTheme.headline6),
+    const PaddedDivider(padding: EdgeInsets.only(bottom: denseRowSpacing)),
+  ];
+}
+
 class DevToolsInfoDialog extends StatelessWidget {
```

Comment:
nit rename class to `DevToolsAboutDialog` to match

Reply:
done